### PR TITLE
feat: #2123 default settlement model

### PR DESCRIPTION
**Pull request by vgenev**
Added support for Default Settlement model

### CODE REVIEW

File: test/integration/settlement_deferred_net_scenario/settlementTransfer.test.js [hunk 1]
```diff
@@ -606,8 +614,68 @@ Test('SettlementTransfer should', async settlementTransferTest => {
...
 
+      test.end()
+    } catch (err) {
+      Logger.error(`settlementTransferTest failed with error - ${err}`)
+      test.fail()
+      test.end()
+    }
+  })
+
+  await settlementTransferTest.test('#66 create a settlement with previous window remaining content and settle it:', async test => {
```
Comment (mdebarros) [inline test/integration/settlement_deferred_net_scenario/settlementTransfer.test.js:625]:
```suggestion
  await settlementTransferTest.test('#66 create a settlement with previous window remaining content and settle it using the DEFAULT_DEFERREDNET Settlement Model:', async test => {
```

File: test/integration/settlement_deferred_net_scenario/settlementTransfer.test.js [hunk 2]
```diff
@@ -75,6 +75,14 @@ const settlementModels = [
     ledgerAccountTypeId: 1, // POSITION
     autoPositionReset: true,
     currencyId: 'USD'
+  },
+  {
+    name: 'DEFERREDNET',
```
Comment (mdebarros) [inline test/integration/settlement_deferred_net_scenario/settlementTransfer.test.js:80]:
```suggestion
    name: 'DEFAULT_DEFERREDNET',
```

**Review by mdebarros — CHANGES_REQUESTED** [review]
Please add the settlementModels[].name to the test names to be more explicit.

For example, see comment on https://github.com/mojaloop/central-settlement/pull/351/files#diff-b7daa6283e695acc3c40fd5b531a3947511767cbaac995e27107e734eee38aeeR625.

See comment about handling the error scenario when no Settlement Model is found for CGS on getSettlementModelByTransferId.

File: src/models/transferSettlement/facade.js [hunk 3]
```diff
@@ -318,7 +318,14 @@ async function getSettlementModelByTransferId (transferId, settlementGranularity
     .where('g.name', settlementGranularityName)
     .where('settlementModel.isActive', 1)
     .select('settlementModel.*')
+  if (settlementModelByTransferId.length === 0) {
+    const allSettlementModels = await Db.from('settlementModel').find()
+    const defaultGrossSettlementModel = allSettlementModels.filter(sm => (sm.currencyId === null && sm.settlementGranularityId === SettlementEnum.SettlementGranularity.GROSS))
+    return defaultGrossSettlementModel
+  }
```
[the remark this answers is not in the document]
Comment (vgenev) [reply src/models/transferSettlement/facade.js:325]: the transfer remains unsettled and should be settled in DEFERREDNET settlement model (either by currency or by default model)

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**